backend/union/policy/eplno.go

```diff
@@ -18,15 +18,28 @@ type EpLno struct {
 	EpAll
 }
 
-func (p *EpLno) lno(upstreams []*upstream.Fs) (*upstream.Fs, error) {
+func (p *EpLno) getUpstreamObjectCount(ctx context.Context, upstream *upstream.Fs) int64 {
+	numObj, err := upstream.GetNumObjects()
+	if err != nil {
+		uName := upstream.Name()
+		fs.LogPrintf(fs.LogLevelNotice, nil,
+			"Number of Objects is not supported for upstream %s, falling back to listing (this may be slower)...", uName)
+		listing, err := upstream.List(ctx, "")
+		if err != nil {
+			fs.LogPrintf(fs.LogLevelError, nil, "Listing failed, treating as 0: %v", err)
```

Comment:
fs.Errorf is the same as fs.LogPrintf(fs.LogLevelError - same for the other debug levels